repository/repository.go

```diff
@@ -462,16 +525,17 @@ func loadData(file string, data interface{}) error {
 type Table string
 
 const (
+	TableLoadBalancers        Table = "loadbalancers"
```

Comment:
Another non-blocker comment, regarding this and all other diffs that show up due to simply moving where code/function is placed in the file: I recommend to keep these out of future feature-PRs, and to use separate (and much simpler and easier to read) PRs that just perform the reorganization of the code placement.

Reply:
Agreed, I will do that in future PRs.